firewall/ipset/operations.go

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2020 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package ipset
+
+import (
+	"net"
+	"strconv"
+)
+
+// SetType defines type of IP set.
+type SetType string
+
+var (
+	// SetTypeHashIP set type uses a hash to store IP addresses where clashing is resolved by storing the clashing elements in an array and, as a last resort, by dynamically growing the hash.
+	SetTypeHashIP = SetType("hash:ip")
+)
+
+// OpVersion is an operation which prints version information.
+func OpVersion() []string {
+	return []string{"version"}
+}
+
+// OpSetCreate is an operation which creates a new set.
+func OpSetCreate(setName string, setType SetType, netMask net.IPMask, hashSize int) []string {
+	args := []string{"create", setName, string(setType)}
+	if netMask != nil {
+		ones, _ := netMask.Size()
+		args = append(args, "--netmask", strconv.Itoa(ones))
```

Comment:
You are setting netmask `/24` for all IPs that added. This will allow exposing more IPs then expected. Just remove this argument.

Reply:
Not passing anymore

Reply:
It's still there. 
It looks like you didn't commit changes but wrote a comment that it's done.

Reply:
I dont pass this optional option, but yes operation still supports it